src/core_modules/capture-core/components/WorkingLists/WorkingListsCommon/helpers/buildFilterQueryArgs/filterConverters/booleanConverter.js

```diff
@@ -12,7 +12,10 @@ export function convertBoolean(filter: BooleanFilterData) {
         values => values.map(filterValue => booleanFilterValues[filterValue]),
         values =>
             (values.length > 1 ?
-                { valueString: values.join(';'), single: false } :
+                {
+                    valueString: values.join(featureAvailable(FEATURES.newAocApiSeparator) ? ',' : ';'),
```

Comment:
Let's roll back this change as breaks the boolean filter in WLs. The `,` is the new separator for UIDs only. Within the `filter`, we use specific formats depending on the filter type and there is no need to change it